doc/user guide.md

```diff
@@ -0,0 +1,346 @@
+  * **"UPNP"**: Enable or disable UPNP forwarding.
+  * **"AddrBook"**: The path to the addrbook.json file local to the datadir ("config/addrbook.json")
+  * **"AddrBookStrict"**: Set true for strict address routability rules, false for local nets.
+  *  "MaxNumInboundPeers": Maximum number of simultaneous peer inbound connections.
+  * **"MaxNumOutboundPeers"**: Maximum number of simultaneous peer outbound connections.
+  * **"FlushThrottleTimeout"**: Time to wait before flushing messages out on the connection in ns
+  * **"MaxPacketMsgPayloadSize"**: Maximum size of a message packet payload, in bytes
+  * **"SendRate"**: Rate at which packets can be sent, in bytes/second
+  * **"RecvRate"**: Rate at which packets can be received, in bytes/second
+  * **"PexReactor"**: Set true to enable the (peer-exchange reactor)[https://docs.tendermint.com/master/spec/reactors/pex/pex.html]
+  * "SeedMode": Is this node a seed_node? (in which node constantly crawls the network and looks for peers. If another node asks it for addresses, it responds and disconnects)
+  * **"PrivatePeerIDs"**: Comma separated list of peer IDs to keep private (will not be gossiped to other peers)
+  * **"AllowDuplicateIP"**: Allow peers with duplicated IP's (according to address book)
+  * **"HandshakeTimeout"**: Timeout in ns of peer handshaking
+  * **"DialTimeout"**: Timeout in ns of peer dialing
+  * **"TestDialFail"**: Testing params. Force dial to fail. Ignore if not testing Tendermint
+  *  **TestFuzz"**: Testing params. FUzz connection. Ignore if not testing Tendermint.
+  * **"TestFuzzConfig"**: Testing params. Fuzz conn config. Ignore if not testing Tendermint.
+
+#### Mempool
+* **"RootDir"**: The data directory of Tendermint's Mempool config (should be the same directory as Pocket Core's data dir)
+* **"Recheck"**: Recheck determines if the mempool rechecks all pending transactions after a block was committed. Once a block is committed, the mempool removes all valid transactions that were successfully included in the block.
+* **"Broadcast"**: Determines whether this node gossips any valid transactions that arrive in mempool. Default is to gossip anything that passes checktx. If this is disabled, transactions are not gossiped, but instead stored locally and added to the next block this node is the proposer.
+* **"WalPath"**: This defines the directory where mempool writes the write-ahead logs. These files can be used to reload unbroadcasted transactions if the node crashes.
+* **"Size"**: MaxSize of mempool in Transactions
+MaxTxsBytes": Max size of ALL Txs in bytes
+* **"CacheSize"**: Max memory cache size of mempool in transactions.
+* **"MaxTxBytes"**: Max size of Tx in bytes
+
+#### FastSync (only if main/fastsync_mode=true)
+Fast Sync version to use:
+* **"Version"**: "v1"
+- "v0" - the legacy fast sync implementation
+- "v1" (default) - refactor of v0 version for better testability
+- "v2" - complete redesign of v0, optimized for testability & readability
```

Comment:
Should we mark v2? as far as I know in our current tendermint fork v2 is non operational.

Reply:
interesting, but actually, the default is still v1